Simplified/NYPLReaderReadiumView.m

```diff
@@ -472,8 +474,35 @@ - (void)webView:(__unused WKWebView *)webView
   }
 }
 
+#pragma mark - ReadiumViewSyncManagerDelegate Methods
```

Comment:
@winniequinn I've attempted to reduce the amount of unnecessary code inside of `ReaderReadiumView.m` to just these three protocol methods. The rest has been pulled out into a separate "Sync Manager" property/class.